src/kibana/components/vislib/visualizations/line_chart.js

```diff
@@ -105,13 +106,23 @@ define(function (require) {
         return color(d.label);
       }
 
+      function showColor(d) {
```

Comment:
The name of this method implies that it returns a boolean, but it does not, it returns the color based on the showCircles option.